### ISSUE TITLE
It doesn't seem to work today

### ISSUE THREAD

**Issue by wisinfun** [wrjlewis/notion-search-alfred-workflow#51]
Search results not found

**Comment by Miaoyifei-mz** [comment]
Same issue.  Have updated cookie and space id but it doesn't work.

**Comment by jonbstrong** [comment]
same here

**Comment by mario-euc** [comment]
Same, right now I'll just share the debug trace for the workflow I'm getting right now.
```
[09:24:24.506] Notion Search[Script Filter] Queuing argument ''
[09:24:24.780] Notion Search[Script Filter] Script with argv '(null)' finished
[09:24:24.783] ERROR: Notion Search[Script Filter] Code 1: Traceback (most recent call last):
  File "/Users/mariomaradiaga/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.BDD3F04A-C769-4E72-AFDF-FAAF14E45C1B/notion.py", line 198, in <module>
    for x in searchResults.pages:
AttributeError: 'Payload' object has no attribute 'pages'
[09:24:24.824] Notion Search[Script Filter] Queuing argument 't'
[09:24:24.865] Notion Search[Script Filter] Queuing argument 'te'
[09:24:25.067] Notion Search[Script Filter] Queuing argument 'tes'
[09:24:25.174] Notion Search[Script Filter] Script with argv '(null)' finished
[09:24:25.176] Notion Search[Script Filter] {"items": [{"uid": 1, "type": "default", "title": "Open Notion - No results, empty query, or error", "arg": "https://www.notion.so/"}]}
[09:24:25.183] Notion Search[Script Filter] Queuing argument 'test'
[09:24:25.451] Notion Search[Script Filter] Script with argv '(null)' finished
[09:24:25.455] Notion Search[Script Filter] {"items": [{"uid": 1, "type": "default", "title": "Open Notion - No results, empty query, or error", "arg": "https://www.notion.so/"}]}
[09:24:25.734] Notion Search[Script Filter] Script with argv '(null)' finished
[09:24:25.739] Notion Search[Script Filter] {"items": [{"uid": 1, "type": "default", "title": "Open Notion - No results, empty query, or error", "arg": "https://www.notion.so/"}]}
```

**Comment by LeeGeunHAHAHA** [comment]
Same here.
I'll share the debug trace.
I couldn't search the page I already have.
![image](https://user-images.githubusercontent.com/21359009/183339561-fa87a2b5-7965-4d17-8a94-f92dff532ec8.png)
![image](https://user-images.githubusercontent.com/21359009/183339599-111ae2c2-71b0-4793-91da-5b4c63d0dfe9.png)

**Comment by amrael** [comment]
Duplicate of #50

**Comment by hongxunming** [comment]
Same here
<img width="509" alt="image" src="https://user-images.githubusercontent.com/43163254/183368416-6486f8c0-d545-47e9-8173-ff7944b9bd95.png">

**Comment by wrjlewis** [comment]
I am on holiday so can't fix this myself for a while

**Comment by hongxunming** [comment]
> Same here <img alt="image" width="509" src="https://user-images.githubusercontent.com/43163254/183368416-6486f8c0-d545-47e9-8173-ff7944b9bd95.png">
It works normally after I changed the notion.py file followed the intro below.
https://github.com/wrjlewis/notion-search-alfred-workflow/issues/50#issuecomment-1209036123

**Comment by wrjlewis** [comment]
This has been fixed with [v0.4.2](https://github.com/wrjlewis/notion-search-alfred-workflow/releases/download/v0.4.2/Notion.Search.alfredworkflow)
